src/games/strategy/triplea/delegate/AbstractUndoableMove.java

```diff
@@ -26,11 +27,33 @@
   protected int m_index;
   protected final Collection<Unit> m_units;
 
+  public AbstractUndoableMove(final Collection<Unit> units) {
+    m_change = new CompositeChange();
+    m_units = units;
+  }
```

Comment:
Would probably be better to have this constructor call the one below with the new CompositeChange() parameter so you only have 'implementation' in one place.